src/TimePicker/TimeDisplay.js

```diff
@@ -70,7 +70,7 @@ class TimeDisplay extends Component {
         borderTopLeftRadius: 2,
         borderTopRightRadius: 2,
         backgroundColor: timePicker.headerColor,
-        color: 'white',
+        color: timePicker.textColor,
```

Comment:
Thanks @oliviertassinari! That makes a lot of sense. I'll update the PR accordingly.